src/v8/remove-passport.ts

```diff
@@ -0,0 +1,27 @@
+import { existsSync } from "fs";
+import { readFile, writeFile } from "fs/promises";
+
+export const stage = "before-install";
+
+export default async function removePassport() {
+    if (!existsSync("api/package.json")) {
+        return;
+    }
+
+    const packageJson = JSON.parse(await readFile("api/package.json", "utf-8"));
```

Comment:
I've added a helper to modify the package.json [here](https://github.com/vivid-planet/comet-upgrade/pull/40), maybe you could use that as well?